octohatrack/__init__.py

```diff
@@ -17,13 +17,17 @@ def main():
 
   repo_name = args.repo_name
 
-  if not repo_exists(repo_name): 
-    print("Repo does not exist: %s" % repo_name)
+  try:
```

Comment:
I'm assuming that this is the only code that makes GH API requests, but its not clear :8ball: